### DISCUSSION TITLE
`input.conf` : Keyboard shortcut to redirect text to a file

### DISCUSSION THREAD

**Discussion by lamyergeier** [mpv-player/mpv#14887] [Q&A]
In input.conf, I did: 

```
ctrl+x run "/bin/bash" "-c" "echo Apple > /mnt/Data/Downloads/.Unsort/temp"; show-text "TEST" 5000
```

On pressing `ctrl+x` The text `TEST` is displayed, but `/mnt/Data/Downloads/.Unsort/temp` is not created.

# Any suggestions?

**Comment by mrfragger** [comment]
ctrl+alt+j    run "[ ! -d ~/Downloads/.Unsort ] && mkdir ~/Downloads/.Unsort ; echo Apple > ~/Downloads/.Unsort/temp ; show-text 'TEST' 5000"

**Reply by lamyergeier** [reply]
@mrfragger  Nothing happens not even the text TEST is displayed with this command. Have you tested it?
